app/models/user.rb

```diff
@@ -138,14 +140,18 @@ def currently_in_exam?
     exams.any? { |e| e.in_progress_for? self }
   end
 
+  def profile_picture
+    (self.avatar || self)&.image_url
```

Comment:
I'll prefer `avatar&.image_url || image_url`
I find that notation quite cumbersome

Reply:
Will change